arm-network/2017-06-01/swagger/azureWAN.json

```
@@ -0,0 +1,2812 @@
+                "user_impersonation"
+            ]
+        }
+    ],
+    "securityDefinitions": {
+        "azure_auth": {
+            "type": "oauth2",
+            "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+            "flow": "implicit",
+            "description": "Azure Active Directory OAuth2 Flow",
+            "scopes": {
+                "user_impersonation": "impersonate your user account"
+            }
+        }
+    },
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroup/{resourceGroupName}/providers/Microsoft.Network/VirtualWANs/{VirtualWANName}": {
+            "get": {
+                "operationId": "VirtualWANs_Get",
+                "x-ms-examples": {
+                    "VirtualWANGet": { "$ref": "../examples/VirtualWANGet.json" }
+                },
+                "description": "Retrieves the details of a VirtualWAN.",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "name": "VirtualWANName",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "The name of the VirtualWAN being retrieved."
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Request successful. Returns the details of the VirtualWAN retrieved.",
+                        "schema": {
+                            "$ref": "#/definitions/VirtualWAN"
+                        }
+                    },
+                    "default": {
```

This should return an error object and the properties must comply with the error definition defined in ARM RPC.